src/test/java/com/hongliangjie/fugue/topicmodeling/LDA/LDATest.java

```diff
@@ -16,6 +18,49 @@
  */
 public class LDATest {
 
+    @Test
+    public void testLoadModel(){
+        URL testFileURL = this.getClass().getClassLoader().getResource("model.ap.json");
+        String testFileName = testFileURL.getPath();
+        Message msg = new Message();
+        msg.setParam("modelFile", testFileName);
+        msg.setParam("multipleModels", 0);
+        msg.setParam("topics", 10);
+        msg.setParam("iters", 50);
+        msg.setParam("saveModel", 0);
+        msg.setParam("random", "native");
+        msg.setParam("exp", 1);
+        msg.setParam("log", 1);
+        LDA lda = new LDA();
+        lda.setMessage(msg);
+        try {
+            lda.loadModel();
+            assertEquals("Load Finished", true, true);
+        }
+        catch (IOException e){
+            assertEquals("Load Finished Failed", true, false);
+        }
+
+    }
+
+    @Test
+    public void testRebuildIndex() throws Exception {
+        Message msg = prepareDocuments();
+        LDA lda = new LDA();
+        lda.setMessage(msg);
+        lda.rebuildIndex();
+        Message returnMsg = lda.getMessage();
+        HashMap<String, Integer> wordsForwardIndex = (HashMap<String, Integer>)returnMsg.getParam("forwardIndex");
+        HashMap<Integer, String> wordsInvertedIndex = (HashMap<Integer, String>)returnMsg.getParam("invertedIndex");
+        assertEquals("Equal Index Size", true, wordsForwardIndex.size() == wordsInvertedIndex.size());
+    }
+
+
+    @Test
+    public void testLikelihood() throws Exception {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/hongliangjie/Fugue/file/2261194273/issues/source?bid=3162994&fileBranchId=3175173#l60)